nextcord/enums.py

```diff
@@ -716,6 +716,11 @@ class AutoModerationActionType(IntEnum):
     timeout = 3
 
 
+class SortOrderTypes(IntEnum):
```

Comment:
This does not match any other enum, where it uses a singular instead of plural names, as there cannot be multiple sort orders at once.